sdk/lib/drivers/wdf/shared/inc/private/common/fxobject.hpp

```diff
@@ -319,7 +319,7 @@ class FxObject {
     union {
         //
         // This field is set when the object is being contructed or before
-        // Commit() and from then on is read only.
+        // Commit() and from then on is read-only.
```

Comment:
This is 3rd-party code, please send the fix upstream.

(cannot find documentation link for it though)